docs/_includes/subscribe.html

```diff
@@ -35,6 +35,19 @@ <h3 class="section-subheading text-muted">{{ site.data.sitetext.subscribe.text |
           </form>
 
         </div>
+
+      </div>
+
+      
+
+    </div>
+    <!--Mailing List-->
+    <div class="mailing-container">
+      <h1>Mailing Lists</h1>
+      <p>Engage in the Service Mesh Performance project. Join any of the mailing lists.</p>
```

Comment:
To keep consistency with the newsletter sub-heading
```suggestion
      <p class="text-muted">Engage in the Service Mesh Performance project. Join any of the mailing lists.</p>
```

Reply:
Yep, I think this sounds better! But Idk why it kind of seems like we're wasting so much space for those two sections. For responsive view, it's good to go as Jubayer said, & for the default preview, @NikhilSharma03 u can experiment with different aesthetic approaches to make it more eye-catching.

Reply:
Thank you for reviewing. I'll make the necessary changes and will get back to you.

Reply:
I couldn't agree more with @Neilblaze.
Please see what more can be done for default.
Good luck 👍